### DISCUSSION TITLE
How to get the extreme ray?

### DISCUSSION THREAD

**Discussion by CruiseSun1998** [yalmip/YALMIP#1260] [General discussion] [answered]
How can I get the extreme ray for an unbounded LP by using YALMIP+Gurobi?

**Comment by johanlofberg** [comment, marked as answer]
If possible via https://yalmip.github.io/Extracting

**Reply by CruiseSun1998** [reply]
Thanks for your answer. I find the extreme ray information in sol.solveroutput.result.unbdray . However, it is a column vector. How can I match it with my decision variables?

**Reply by johanlofberg** [reply]
That you will have to find out via reverse engineering, i.e. do it or simple problems where you understand what it should be etc

**Reply by CruiseSun1998** [reply]
Well, thanks a lot. I will try for it. Thank you again!
